src/main/java/org/springframework/data/elasticsearch/core/query/NativeSearchQuery.java

```diff
@@ -40,6 +40,7 @@
 	private List<FacetRequest> facets;
 	private List<AbstractAggregationBuilder> aggregations;
 	private HighlightBuilder.Field[] highlightFields;
+	private IndexBoost[] indicesBoost;

```

Comment:
IMO, the `List<>` member fields are more user friendly than the array/varargs ones, i.e. it is more easier at the use-side to add/remove entries.